docs/webhooks/Testing.md

```diff
@@ -0,0 +1,24 @@
+# Testing
+
+To enable you to check you have set up your infrastructure correctly, we offer a test event type. This can be triggered via our portal or API and does not contain any information pertaining to a resource, just the following:
```

Comment:
should we specify exactly where you can trigger the test webhook? ie, in the webhooks subscriptions page in admin portal, or even add a screenshot?